frontend/src/Teacher/InbusImport.vue

```diff
@@ -0,0 +1,279 @@
+<script setup lang="ts">
+/**
+ * This component provides import of classes of students from Edison
+ * into Kelvin using the INBUS API.
+ *
+ * It is only available to teachers, and it is accessible from the main page
+ * next to class selection drop down menu.
+ */
+import { computed, onMounted, ref } from 'vue';
+
+import { csrfToken } from '../api.js';
+import { ConcreteActivity, InbusSubjectVersion } from './inbusdto';
+
+interface KelvinSubject {
+  name: string;
+  abbr: string;
+}
+
+interface Semester {
+  pk: number;
+  year: number;
+  winter: boolean;
+}
+
+interface ImportResult {
+  login: string;
+  firstname: string;
+  lastname: string;
+  created: boolean;
+}
+
+interface Result {
+  users: ImportResult[];
+  count: number;
+  error?: string;
+}
+
+let subject_inbus_selected = ref<InbusSubjectVersion | null>(null);
```

Comment:
`subject_inbus_selected` is not needed, you can just read the selected value from a parameter of `loadScheduleForSubjectVersionId`. There's not really any point in combining both `@change` and `v-model`.

Reply:
OK. Now I have to use index into an array that I don't like but we can iterate on that later on.